app/models/reference_feature_metrics.rb

```diff
@@ -36,6 +36,7 @@ def percentage_references_within(
 
   def time_to_get_references(start_time, end_time = Time.zone.now)
     applications = ApplicationForm
+      .apply_1
```

Comment:
I think we should be checking for `duplicate: false` instead here as some people request new references as part of their apply_again application. Sorry i should have picked this up as part of my earlier PR

Reply:
where('"references".feedback_provided_at BETWEEN ? AND ? AND duplicate = ? ', start_time, end_time, false)``

Reply:
I vaguely remember talking to someone about whether including apply again applications would skew the stats. The problem is that some apply again applications will have no new references, some might update one, others update both and to compare like with like you only really want to consider the ones that are replacing both references. I think your suggestion achieves that though.